parquet-column/src/main/java/org/apache/parquet/column/ParquetProperties.java

```diff
@@ -50,6 +50,7 @@
   public static final boolean DEFAULT_ESTIMATE_ROW_COUNT_FOR_PAGE_SIZE_CHECK = true;
   public static final int DEFAULT_MINIMUM_RECORD_COUNT_FOR_CHECK = 100;
   public static final int DEFAULT_MAXIMUM_RECORD_COUNT_FOR_CHECK = 10000;
+  public static final int DEFAULT_MAXIMUM_BLOOM_FILTER_BYTES = 16 * 1024 * 1024;
```

Comment:
The default bloom filter max is 1/8th of the default row group size?

Reply:
This need to discuss.